templates/emails/thankyou.hbs

```diff
@@ -29,7 +29,7 @@ Subject: Thank you for your contribution to {{collective.name}}
 
 <p>A message from {{collective.name}}:</p>
 
-<p>{{{customMessage}}}</p>
+<div style="color: #494B4D; font-size: 14px; line-height: 18px; padding-left: 10px; border-left: 5px solid #F1F2F3">{{{customMessage}}}</div>
```

Comment:
In a recent review you told me there was no need to put a condtion before displaying `customMessage` because it would be invisible anyway. This is not the case anymore with this change.

Reply:
ops, sorry I seem to have missed this condition. 😢 I've added this.